### ISSUE TITLE
Need more device information

### ISSUE THREAD

**Issue by maxroberts** [maxroberts/interfacing-to-hospital-equipment#4]
Need people to reach out to frontline doctors/nurses they know to ask about:
- Devices that they are having issues with (name, make, model)
- Specifics of the issue

Ask what problem they are having, not necessarily what they explicitly want fixed. If we understand the problem, we might be able to look more generally for solutions.

**Comment by kroylar** [comment]
@maxroberts yes, this is very important. In addition I would like to know what they are currently doing to address their problem. They may thing this is obvious but to us its not, so its important we document it.
